eng/common/scripts/Write-FileSystemMetrics.ps1

```diff
@@ -0,0 +1,29 @@
+foreach($volume in Get-Volume) {
```

Comment:
Do you plan to add cross-platform disk metric support?

Reply:
This should be cross-platform. Is there something you think should not work?

Reply:
I didn't realize Get-Volume was windows only.   Looking at `df` for linux

Reply:
Added using .net core's `[IO.DriveInfo]::GetDrives()`